server/ERNI.PBA.Server.Host/Exceptions/OperationErrorException.cs

```
@@ -1,21 +1,22 @@
 ﻿using System;
+using Microsoft.AspNetCore.Http;
 
 namespace ERNI.PBA.Server.Host.Exceptions
 {
     public class OperationErrorException : Exception
     {
         public OperationErrorException(string message)
-            : base(message)
+            : this(StatusCodes.Status400BadRequest, message)
```

this default value is imho misleading